### PR TITLE
JBIDE-22912 - Deploy Docker Wizard: Inappropriate project combo size

### PR DESCRIPTION
**Pull request by snjeza**
https://issues.jboss.org/browse/JBIDE-22912
Deploy Docker Wizard: Inappropriate project combo size

### CODE REVIEW

**Comment by adietish** [comment]
@mlabuda do you have FC24/GTK 3.20 (I dont, I am running FC23/GTK 3.18). If you have, can you please verify this?

**Comment by bdshadow** [comment]
Verified the patch, but combo box still has inappropriate size.

gtk3-3.20.9-1.fc24.x86_64

**Comment by bdshadow** [comment]
@snjeza as i remember the fix is alright now. Could you, please, rebase the PR so that we could merge it?

**Comment by snjeza** [comment]
I have rebased it.

File: plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/deployimage/DeployImagePage.java [hunk 1]
```diff
@@ -183,6 +185,20 @@ protected void onPageWillGetActivated(final Direction progress, final PageChangi
...
+		// https://issues.jboss.org/browse/JBIDE-22912,
+		// https://bugs.eclipse.org/bugs/show_bug.cgi?id=500703
+		getControl().addPaintListener(new PaintListener() {
+
+			@Override
+			public void paintControl(PaintEvent e) {
+				if (getControl().isVisible()) {
+					((Composite) getControl()).layout(true, true);
+					((Composite) getControl()).update();
+					getControl().getShell().pack();
```
Comment (bdshadow) [inline plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/deployimage/DeployImagePage.java]: @snjeza , why is it packing here and no the same one in AbstractProjectPage ?

**Comment by snjeza** [comment]
I have made only necessary changes.

**Pull request merged**